devops/actions/llvm_test_suite/action.yml

```diff
@@ -83,12 +88,11 @@ runs:
       if [ -e /runtimes/oneapi-tbb/env/vars.sh ]; then
         source /runtimes/oneapi-tbb/env/vars.sh;
       fi
+      echo $LD_LIBRARY_PATH
```

Comment:
Please, add a description for this command.
It looks like a list of random directories. It should be clear from the log what is printed w/o looking at script sources.